Jenkinsfile

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env groovy
+
+REPOSITORY = 'release'
+DEFAULT_SCHEMA_BRANCH = 'deployed-to-production'
```

Comment:
[It doesn't look like this app includes schema tests](https://github.com/alphagov/release/blob/master/jenkins.sh) so this and other schema related config isn't necessary.